python/test/unit/fem/test_expression_evaluation.py

```diff
@@ -97,3 +93,66 @@ def grad_expr1(x):
     b2.interpolate(grad_expr1)
 
     assert np.isclose((b2.vector - b.vector).norm(), 0.0)
+
+
+def test_simple_evaluation():
+    """Test evaluation of UFL Expression.
+
+    This test evaluates a UFL Expression on two cells in the mesh and compares the
+    result with an analytical expression.
+
+    For a donor function f(x, y) = (x^2 + 2*y^2) result is compared with the
```

Comment:
What is a 'donor' function?

Reply:
An input function to the evaluation. The expression represents pointwise gradient() operator, so it maps input functions to output function. In different terminology, it maps a donor function to a target function. It is a donor in a sense that is provides expansion coefficients to the kernel. We can change it to "input function" as well.